b_cfn_sagemaker_endpoint_tests/integration/tests/sagemaker_endpoint_test.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+from b_aws_testing_framework.credentials import Credentials
+
+from b_cfn_sagemaker_endpoint_tests.integration.infrastructure import Infrastructure
+
+
+def test_INFRASTRUCTURE_sagemaker_endpoint_WITH_invocation_EXPECT_success_message():
```

Comment:
You forgot to test the most important part - to test that message is "Everything's Gucci! <3", then update S3 code, wait for deployment to finish and run again to expect different message.

Reply:
And also, I would love to see more tests. Can you think of more edge / test cases?

Reply:
It's currently in progress